types/src/v2/error.rs

```diff
@@ -121,7 +126,7 @@ impl JsonRpcErrorCode {
 	}
 
 	/// Returns the message for the given error code.
-	pub const fn message(&self) -> &str {
+	pub const fn message(&self) -> &'static str {
```

Comment:
I guess a `&str` returned from a `const fn` is `'static` by definition, but I don't mind being explicit. 👍

Reply:
It didn't work in the `From impl` so I had put `'static` there.

```
impl<'a> From<JsonRpcErrorCode> for JsonRpcErrorObject<'a> {
	fn from(code: JsonRpcErrorCode) -> Self {
		Self { code, message: code.message(), data: None }
	}
}
```

Not sure if there's a better work around for that